examples/flutter_gallery/README.md

```diff
@@ -1,6 +1,25 @@
 # Flutter gallery
 
-Demo app for the Material Design widgets and other features provided by Flutter.
+Demo app for the material design widgets and other features provided by Flutter.
+
+## Building
+
+You can follow these instructions to build the gallery app
+and install it onto your device.
+
+### Prereqs
+
+If you are new to Flutter, please first follow
+the [Flutter Setup](https://flutter.io/setup/) guide.
+
+### Building and installing the Flutter app
+
+* `cd $FLUTTER_ROOT/examples/flutter_gallery`
+* `flutter upgrade`
+* `flutter devices` and note the name of the device you want to build for
```

Comment:
Instead of adding this line, I'd make sure that `flutter run` prints a reasonable error/instructions for when it finds more than one device.  I doubt having multiple devices connected is a common case.